pyark/__init__.py

```diff
@@ -457,6 +459,10 @@ def main(argv=None):
     ch.setFormatter(log_formatter)
     logger.addHandler(ch)
 
+    if not args.apipassword:
+        args.apipassword = getpass("Please enter password for {}: "
+                                   .format(args.apiuser))
+    
```

Comment:
Please remove the whitespaces on line 465 and make flake8 / Travis green again. :)